ra_aid/agent_utils.py

```
@@ -57,7 +57,10 @@
 from ra_aid.models_params import (
     DEFAULT_TOKEN_LIMIT,
 )
-from ra_aid.utils.agent_thread_manager import agent_thread_registry, has_received_stop_signal
+from ra_aid.utils.agent_thread_manager import (
+    agent_thread_registry,
```

Ruff (0.11.9)

`ra_aid.utils.agent_thread_manager.agent_thread_registry` imported but unused

Remove unused import: `ra_aid.utils.agent_thread_manager.agent_thread_registry`

(F401)